src/features/config/index.js

```diff
@@ -0,0 +1,12 @@
+
+export default {
+    /**
+     * Application name.
+     */
+    appName: 'Jitsi Meet',
+
+    /**
+     * The domain of the Jitsi Meet deployment that will be used.
+     */
+    defaultDomain: 'meet.jit.si'
+};
```

Comment:
Maybe this file should be better as src/config/index.js.
WDYT?

Reply:
I'm following the same pattern as jitsi-meet here. Our idea of a feature is an abstraction of certain functionality, which other features can then use / consume. In that line, the "config" feature exposes some defaults to other features. Also, when redux is added, this will grow to store the user set display name, server URL, etc.

Reply:
Hmm... I see.